QuantConnect.AlphaVantage/AlphaVantageDataDownloader.cs

```diff
@@ -125,7 +125,7 @@ public IEnumerable<BaseData> Get(DataDownloaderGetParameters dataDownloaderGetPa
             if (endUtc < startUtc)
             {
                 Log.Error($"{nameof(AlphaVantageDataDownloader)}.{nameof(Get)}:InvalidDateRange. The history request start date must precede the end date, no history returned");
-                return Enumerable.Empty<BaseData>();
```

Comment:
The period is invalid so the history should be empty, it's a different case

Reply:
https://github.com/QuantConnect/Lean/pull/7812